Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml

```diff
@@ -34,21 +38,26 @@
   <display>
     <vis name="hcal_steel_vis" r="0.5" g="0.5" b="0.5" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
     <vis name="hcal_steel_master_vis" r="0.6" g="0.6" b="0.6" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
+    <vis name="hcal_steel_support_vis" r="0.8" g="0.3" b="0.3" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
     <vis name="hcal_face_plate_vis" r="0.8" g="0.3" b="0.3" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
     <vis name="hcal_end_plate_vis" r="0.8" g="0.3" b="0.3" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
     <vis name="hcal_polystyrene_vis" r="0.8" g="0.3" b="0.3" alpha="1" showDaughters="true" visible="true" drawingStyle="solid" />
     <vis name="hcal_envelope" r="0.6" g="0.8" b="0.6" alpha="1" showDaughers="true" visible="true" />
     <vis name="hcal_barrel_seq1_vis" r="0.1" g="0.8" b="0.6" alpha="1" showDaughers="true" visible="true" />
     <vis name="hcal_barrel_seq2_vis" r="0.1" g="0.6" b="0.8" alpha="1" showDaughers="true" visible="true" />
+    <vis name="hcal_barrel_layer_vis" r="0.1" g="0.6" b="0.8" alpha="1" showDaughers="true" visible="true" />
     <vis name="hcal_air_vis" r="1" g="1" b="1" alpha="1" showDaughers="true" visible="true" />
   </display>
 
   <readouts>
     <readout name="BarHCal_Readout">
       <!-- Introduce a segmentation in phi to devide HCAL modules, check for advantages of finer granularity -->
      <segmentation type="GridPhiEta" grid_size_eta="100" phi_bins="256"/>
-     <id>system:4,layer:6,module:7,row:10,tile:21,eta:1,phi:-9</id>
-
+     <id>system:4,module:7,row:9,layer:5,tile:2,eta:1,phi:-9</id>
+    </readout>
+    <readout name="BarHCal_Readout_phieta">
+     <segmentation type="GridPhiEta" grid_size_eta="0.01" phi_bins="256"/>
```

Comment:
As discussed earlier.. it's not a misprint, it follows the segmentation used in the ECal to allow a smoother combined reconstruction. However, this segmentation is not used for the "standard" recon in the HCal. For now, it should remain and can be adjusted later, when the combined reco has been studied.